pandas/tests/io/test_excel.py

```diff
@@ -315,18 +316,17 @@ def test_excel_table(self):
 
     def test_reader_special_dtypes(self):
 
-        expected = DataFrame.from_items([
-            ("IntCol", [1, 2, -3, 4, 0]),
-            ("FloatCol", [1.25, 2.25, 1.83, 1.92, 0.0000000005]),
-            ("BoolCol", [True, False, True, True, False]),
-            ("StrCol", [1, 2, 3, 4, 5]),
+        expected = DataFrame.from_dict(OrderedDict({
```

Comment:
I need to change the contents of the OrderedDict from a dict to a list of tuples to ensure that order is maintained to stop the tests failing